.semaphore/semaphore.yml

```diff
@@ -143,6 +144,14 @@ blocks:
           commands:
             - make fv
             - make dirty-check
+            - '[[ -f ./report/fv/fv_suite.xml ]] && test-results publish ./report/fv/fv_suite.xml'
```

Comment:
It might be worth it to just publish the results of `./report/fv`, same as you're doing for `./report/fv`, just in case we get a little more robust with our FVs.